src/odemis/util/test/spot_test.py

```diff
@@ -92,21 +95,28 @@ def test_precomputed(self):
 
 class TestFindCenterCoordinates(unittest.TestCase):
     """
-    Test FindCenterCoordinates()
+    Unit test class to test the behavior of FindCenterCoordinates in
+    odemis.util.spot.
     """
+
+    imgfile = TIFF.open('spotdata.tif', mode='r')
+    imgdata = numpy.array(list(imgfile.iter_images()))
+    imgfile.close()
+    coords0 = numpy.genfromtxt('spotdata.csv', delimiter=',')
+
     def test_find_center(self):
         """
         Test FindCenterCoordinates
         """
         expected_coordinates = [(-0.00019439548586790034, -0.023174120210179554),
-                                (0.41813787193469681, -0.77556146879261101),
+                                (0.47957790544657719, -0.82786251901339769),
```

Comment:
I had to update these values because the FindCenterCoordinates changed. I plotted the images with the old and new values and they looked pretty much the same.